src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC } from 'react';
+
+interface Props {
+  setHasError: (boolean: boolean) => void
+}
+
+export const ErrorNotification:FC<Props> = ({ setHasError }) => {
```

Comment:
Try to avoid using names for props the way you use them for useState. Sometimes this is confusing and sounds like a hard binding.
```suggestion
export const ErrorNotification:FC<Props> = ({ onChangeError }) => {
```